docs/src/content/docs/resources/community-content.mdx

```diff
@@ -74,7 +74,11 @@ Explore community-produced content maintained by Starlight users:
 		title="Publishing Documentation with Astro Starlight"
 		description="Getting started with Starlight documentation"
 	/>
-
+<LinkCard
+		href="https://events-3bg.pages.dev/jotter/starlight/guide/"
+		title="Enable View Transitions"
+		description="Leverage the <span style='filter: grayscale(100%)'>👜 bag of tricks ✨</span> for that SPA-like look and feel along with cool view transitions"
```

Comment:
The `<span>` here interferes with the clickable area of the link, so it might be better to use a simple text-only version of the description like this:

```suggestion
		description="Get that SPA-like look and feel with the Bag of Tricks’ view transitions support"
```